TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/objects/Proxy.java

```diff
@@ -295,12 +296,12 @@ public boolean isConnected() {
         return connected;
     }
 
-    public DnsRecord getDnsRecord() {
-        return dnsRecord;
+    public ArrayList<DnsRecord> getDnsRecord() {
```

Comment:
Change to `public Collection<DnsRecord> getDnsRecords()`

Reply:
Why do we need multiple DNS records?

Reply:
wenn man 2 subdomains haben will.

Reply:
wieso public, wir haben eine get methode.

Reply:
Ja die get Method soll auch bleiben, aber sie soll eine Collection<DnsRecord> zurückgeben und getDnsRecord**s** heißen ;-)